sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamSource.scala

```diff
@@ -126,7 +132,12 @@ class FileStreamSource(
       unreadFiles
     } else {
       // All the new files found - ignore aged files and files that we have seen.
```

Comment:
Should this comment follow `fetchAllFiles`?

Reply:
Will update the comment here to describe the logic for Trigger.AvailableNow instead.